TipCatDotNet.Api/Models/HospitalityFacilities/MemberResponse.cs

```diff
@@ -25,26 +27,28 @@ public MemberResponse(int id, int? accountId, string firstName, string lastName,
         [Required]
         public string LastName { get; }
         public string? Email { get; }
+        public string MemberCode { get; }
```

Comment:
Add the required attribute to show the field is non-nullable in Swagger. That's important for the front-end.